pkg/operator/ceph/object/topic/provisioner.go

```diff
@@ -142,17 +138,12 @@ func createSNSClient(sess *awssession.Session) (snsClient *sns.SNS) {
 			}
 		},
 	})
-	return
+	return snsClient, nil
 }
 
-func (p *Provisioner) Create(topic *cephv1.CephBucketTopic) (*string, error) {
+var Create = func(p *Provisioner, topic *cephv1.CephBucketTopic) (*string, error) {
```

Comment:
Do we really need that pointer to a string in the return?

Also why is `p` now a pointer? Typically, if it's a small struct and we don't need to mutate it, it shouldn't be a pointer.

Reply:
> Do we really need that pointer to a string in the return?

the ARN in the `CreateTopicOutput` (an AWS SDK struct) is a `*string`, so I'm just returning the value from there